lib/explorer/backend/lazy_series.ex

```diff
@@ -47,6 +40,21 @@ defmodule Explorer.Backend.LazySeries do
     window_mean: 5,
     window_min: 5,
     window_sum: 5,
+    # Transformation
+    cast: 2,
+    column: 1,
+    reverse: 1,
+    argsort: 2,
+    sort: 2,
+    distinct: 1,
+    unordered_distinct: 1,
+    slice: 3,
+    head: 2,
+    tail: 2,
+    peaks: 2,
+    fill_missing: 2,
+    fill_missing_with_value: 2,
+    coalesce: 2,
```

Comment:
I think coalesce, cast, fill_missing and fill_missing_eith_value are all element wise because they consider each element in isolation, right?

Reply:
Actually, fill missing is window. With value is element wise.

Reply:
This makes me wonder if we could have this functionality splitted. Today is only one function that accepts both. I was thinking maybe we should have `Series.fill_missing/2` with the strategy, and `Series.fill_missing_with_value/2` for the variation with value. WDYT?